doc/develop/extending.rst

```diff
@@ -175,9 +175,9 @@ Creating the Odemis environment
 
 Install `Anaconda <https://www.anaconda.com/distribution/>`_ with Python 2.7 32 bits.
 It's also possible to use the version with Python 3.7, 32 or 64 bits, in which case
```

Comment:
Shouldn't this line:
"conda create -y --name odemisdev python==3.6.8" @ linenumber 202
Also be updated/changed to python 3.7?

Reply:
As described in my previous response, that's actually on purpose, to change the Python interpreter to a version for which Odemis has been well tested.